packages/astro/src/integration/namespace.ts

```diff
@@ -0,0 +1,26 @@
+import module from 'module';
```

Comment:
Also let's use the named import here which should allow us to get rid of the tsconfig

Reply:
It doesn't work. That's why I had to change the tsconfig. Here's the full error message:

> src/integration/namespace.ts(1,32): error TS2497: This module can only be referenced with ECMAScript imports/exports by turning on the 'allowSyntheticDefaultImports' flag and referencing its default export.